projects/7b-crypto/project.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 7B Crypto
+description: A crypto broker app for people who want to trade with an easy-to-use interface, suited to both beginners and advanced traders. 
+logo: False
```

Comment:
@AmritKumar @renlulu had a discussion with Diane and we can omit the logo entirely now. I wrote a script that will automatically detect the logo in the folder and display it.